files/grest/rpc/stake_address/stake_distribution.sql

```diff
@@ -0,0 +1,106 @@
+CREATE TABLE IF NOT EXISTS GREST.STAKE_DISTRIBUTION (
```

Comment:
I think this will likely be a trigger eventually or a cron-query, so we prolly might wanna move it to cache-queries folder

Reply:
Yep, I was thinking the same. I put it in here because everyone can just copy and run it without worrying about anything for testing phase.

I think the best option is to move the creation of ALL tables we want to eventually create in our `grest` schema to be a part of the `grest-setup.sh` script.

Reply:
Hmm, given that this will run multiple times at scheduled intervals, I feel it can stay as a scheduled query, regardless of state of setup script. The setup script would download these cache queries and schedule them if needed

Reply:
Ah, I see what you mean now... I thought you were referring to the stake_distribution query as the scheduled one.

Yeah I think 2 sets of scheduled queries make sense:
1. grest schema setup queries -> ensures we always have correct structure even if we drop and recreate db
2. data queries that populate grest schema like the one below

Did I understand correctly?

Reply:
I originally meant having both as part of same (since it wouldnt be recreating if one already exists)..but yea both seperately works too